webpages/js/metaanalyses.js

```diff
@@ -1478,6 +1489,16 @@
       });
     });
 
+    // Add unhide button for the needed papers
```

Comment:
This is cool, but we might as well do this while looping originally? Further up, at line 1340 we already get the number of unhidden experiments.

We can compare this number to the length of paper.experiments (i.e. if unhidden is less than total, we must have hidden experiments), saving ourselves from needing to loop again, or have another function called getNbHidden.

We also already have paperTitleEl, so I think it makes sense to move this above